recipes/atactk/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.6" %}
+
+package:
+    name: atactk
+    version: {{ version }}
+
+source:
+    fn: atactk-{{ version }}.tar.gz
+    url: https://github.com/ParkerLab/atactk/archive/{{ version }}.tar.gz
+    sha256: 887363209d556ee5fb4969f39bab1b747cca56b6bd1c0fd9e48d69208e7a5f74
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - pysam
+        - python-levenshtein
+        - sexpdata
+
+test:
+    commands:
+        - trim_adapters --help
+        - make_cut_matrix --help
+
+extra:
+    recipe-maintainers: 
+        - egeogretmen
+    notes:
+        - Adds 3 scripts, namely "trim_adapters", "make_cut_matrix" and "plot_aggregate_matrix.R"
```

Comment:
test all 3 scripts

Reply:
Is there a way to test Rscripts easily? They have to be explicitly coded to print a help text I guess. How can I test [this](https://github.com/ParkerLab/atactk/blob/master/scripts/plot_aggregate_matrix.R) Rscript?

Reply:
Just import the R modules: RColorBrewer, ggplot2, grid, gtools. That should be enough, you also need to depend on this as runtime dependency.

See here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/r-gbm/meta.yaml#L37